output/out_logzio_test.go

```diff
@@ -23,7 +23,7 @@ func TestSerializeRecord(test *testing.T) {
 	record["five"] = 5
 	testServer := time.Now()
 
-	serialize, err := serializeRecord(testServer, "atag", record, "logzio", defaultId)
+	serialize, err := serializeRecord(testServer, "atag", record, "logzio", defaultId, false, false, "_")
```

Comment:
Why do we need to change the parameters here?

Reply:
I changed the `serializeRecord` function call in the tests to include additional parameters (`dedotEnabled`, `dedotNested`, `dedotNewSeperator`) to match the updated function signature. The tests were initially broken due to missing these parameters, and I fixed them by aligning with the new definition. These parameters control dedotting behavior, ensuring field names are appropriately processed.